bin/configurable-http-proxy

```diff
@@ -294,6 +298,24 @@ if (!options.authToken) {
   log.warn("REST API is not authenticated.");
 }
 
+// custom headers option
+options.customHeaders = [];
+if (args.customHeaders) {
+  options.customHeaders = args.customHeaders.toString().split(',').map(s => s.trim()).map(header => {
+    var i = header.indexOf(':');
+    var key, value;
+    if (i < 0) {
+      key = header;
```

Comment:
Probably error here would be more useful than assuming an empty string for invalid input?

Reply:
Implemented.